onnx/backend/test/case/node/__init__.py

```diff
@@ -102,6 +102,13 @@ def function_testcase_helper(node, name):  # type: (NodeProto, Text) -> List[Nod
 
 
 def _extract_value_info(arr, name):  # type: (np.ndarray, Text) -> onnx.ValueInfoProto
+    if arr.dtype == "object":
```

Comment:
What happens here if "arr" is a list, instead of an array? (As an aside, the type-annotations above will no longer be valid. Unfortunately, it looks like the python type-checker is not part of the CI ... this is a separate issue, but it does complicate our task here since we can't rely on the type-checker.) I don't understand how things can work without changes here.

Reply:
I've just addressed this change as part of the recent updates for arr->list, but some work might need to be done in sequence_value_info as well. It seems like it's only used in the shape-inference-tests: https://github.com/onnx/onnx/blob/8e9c7d57f7c5aa6f6eb7ee7abb0ba2a243781933/onnx/test/shape_inference_test.py#L2522